packages/babel-parser/src/parser/expression.js

```diff
@@ -482,28 +480,28 @@ export default class ExpressionParser extends LValParser {
   }
 
   // Parse unary operators, both prefix and postfix.
-
-  parseMaybeUnary(refExpressionErrors: ?ExpressionErrors): N.Expression {
+  // https://tc39.es/ecma262/#prod-UnaryExpression
+  parseUnary(refExpressionErrors: ?ExpressionErrors): N.Expression {
```

Comment:
I think that this was called `parseMaybeUnary` (like `parseMaybeConditional`, `parseMaybeAssign`) because it could return something which is not a unary expression, like a simple identifier.

I know that _technically_ Identifier is a child of UnaryExpression in the spec, but it's just a side effect of how it's specified.